### PR TITLE
instrumantation.mod: add logs

### PR DESCRIPTION
**Pull request by piotrdanio**
Add logs for instrumentation.md

### CODE REVIEW

**Comment by acwiek** [comment]
remove html tags

File: docs/instrumentation.md [hunk 1]
```diff
@@ -292,4 +292,19 @@ tpm_tis_spi.ko: tpm_tis_spi_probe()
 tpm_tis_spi.ko: tpm_tis_spi_driver_probe()
 tpm.ko: tpm_dev_release()
 tpm.ko: tpm_dev_release()
+
+```
+
+### Whole logs from the test procedure
```
Comment (acwiek) [inline docs/instrumentation.md:298]: maybe replace with something like that 

### Whole logs from the test procedure

Below we have collected call stacks for Raspberry Pi TPM driver startup.

[Log file with TPM module plugged in](./logs_tpm_module/log_with_tpm_get_call_stack.txt)</br>
[Log file without TPM](./logs_tpm_module/log_without_tpm_get_call_stack.txt)</br>

File: docs/instrumentation.md [hunk 2]
```diff
@@ -292,4 +292,19 @@ tpm_tis_spi.ko: tpm_tis_spi_probe()
...
+terminal window by default. </br>
+```concole
+$ ./pull_logs.sh
+```
+[Log file with  or without TPM module plugged in](./logs_tpm_module/log_with_tpm_pull_logs.txt)</br>
+```concole
+$ python get_call_stack.py 
 ```
+[Log file with TPM module plugged in](./logs_tpm_module/log_with_tpm_get_call_stack.txt)</br>
+[Log file without TPM](./logs_tpm_module/log_without_tpm_get_call_stack.txt)</br>
```
Comment (acwiek) [inline docs/instrumentation.md:310]: remove html tags

File: docs/logs_tpm_module/log_with_tpm_pull_logs.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+RASPI_IP: pi@192.168.4.44
```
Comment (acwiek) [inline docs/logs_tpm_module/log_with_tpm_pull_logs.txt:1]: this file isn't needed

File: docs/instrumentation.md (hunk 1 above)
Comment (ogtomi) [reply docs/instrumentation.md:298]: Changes applied in commit `0acd479aaffb4ea534d45381fa4415344b7cced6`

File: docs/instrumentation.md (hunk 2 above)
Comment (ogtomi) [reply docs/instrumentation.md]: HTML tags removed in commit `0acd479aaffb4ea534d45381fa4415344b7cced6`

File: docs/logs_tpm_module/log_with_tpm_pull_logs.txt (hunk 3 above)
Comment (ogtomi) [reply docs/logs_tpm_module/log_with_tpm_pull_logs.txt]: File deleted in commit `d552c2eaccbf5fc92778426c5f66aa93e54ace49`

**Pull request merged**